apps/system/test/unit/statusbar_test.js

```diff
@@ -105,7 +105,7 @@ suite('system/Statusbar', function() {
       // executing init again
       Statusbar.start();
       Statusbar.finishInit();
-      Statusbar._paused = 0;
+      Statusbar._numMinimizedIcons = 0;
```

Comment:
instead of touching the private variable can you do the `defineProperty` dance [1] instead to control the computed value?

This will cover the `calculateMinized` code and make it much easier to evolve in the future.

[1] https://github.com/mozilla-b2g/gaia/blob/cf5f2a5bf41d2db088f7f5ade5dfa4c01f91b6f2/apps/system/test/unit/notification_screen_test.js#L94-L97